facility-all-programs.ipynb

```diff
@@ -318,7 +466,7 @@
     "        print( \"Data from RCRA_EVALUATIONS stored in rcra_insp_data.\")\n",
     "        have_rcra_insp = True\n",
     "    except pd.errors.EmptyDataError:\n",
-    "        print( \"No data for this facility in RCRA_EVALUATIONS.\")\n",
+    "        print( \"This is a RCRA-regulated facility, but there's no data for it in RCRA_EVALUATIONS.\")\n",
```

Comment:
This is a really helpful change, I wondered about this. Do we also have a case for "this facility is not regulated under RCRA"?

Reply:
done! And for the other programs.